extra/bookmarks.vim

```diff
@@ -1,25 +1,33 @@
 " List bookmarks
 " Create bookmark 0 with <C-S-0>, Create bookmark 1 with <C-S-1> and so on.
 " Go to bookmark 0 with <C-0>, go to bookmark 1 with <C-1> and so on.
+" Show bookmarks
+let g:WhichKeyDesc_Bookmarks_Show = "<leader>Bl show-bookmarks"
```

Comment:
```suggestion
let g:WhichKeyDesc_Bookmarks_List = "<leader>Bl list-bookmarks"
```
The name of the command is present on line 1.
I prefer "list" instead of "show", because it has the initial `L`.
Do you agree?

Reply:
Completly